pdns/pdns_recursor.cc

```diff
@@ -1426,7 +1462,17 @@ static void startDoResolve(void *p)
       pbMessage->setQueryTime(dc->d_now.tv_sec, dc->d_now.tv_usec);
       pbMessage->setRequestorId(dq.requestorId);
       pbMessage->setDeviceId(dq.deviceId);
+#ifdef NOD_ENABLED
+      if (g_nodEnabled) {
+        if (nodCheckNewDomain(dc->d_mdp.d_qname))
```

Comment:
It looks like you won't have `NOD` without `protobuf`, I don't think you want that!

Reply:
No, indeed!